src/test/java/uk/ac/ebi/eva/test/utils/DropStudyJobTestUtils.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2015-2017 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.ac.ebi.eva.test.utils;
+
+import com.mongodb.BasicDBObject;
+import com.mongodb.DBCollection;
+
+import static org.junit.Assert.assertEquals;
+import static uk.ac.ebi.eva.commons.models.converters.data.VariantToDBObjectConverter.FILES_FIELD;
+import static uk.ac.ebi.eva.commons.models.converters.data.VariantToDBObjectConverter.STATS_FIELD;
+import static uk.ac.ebi.eva.commons.models.data.VariantSourceEntity.STUDYID_FIELD;
+
+public class DropStudyJobTestUtils {
```

Comment:
I think jmmut has a point here and the code seems clear to me, it seems that is there are future issues there i can grasp the meaning quickly even if the assert functions. The only improvement would change the name to assert instead of check to reassure its purpose.